### DISCUSSION TITLE
Move mouse to only one screen in multimonitor setups when switching from client to server

### DISCUSSION THREAD

**Discussion by alepouna** [input-leap/input-leap#1943] [Q&A]
I have been searching for a while but maybe my queries are incorrect, I am trying to figure out how to have another client device not throw the mouse on other screens. I currently run a triple monitor setup, and an external client to be 'below' my main (center) monitor. 

Currently, when the mouse is moved upwards from the client, depending on the position it will go to one of the 3 available screens. Is there a way to only have the mouse go to the center screen?

**Comment by waregle82** [comment]
Same setup for me. client is under server running 3 monitors. Temp solution is to put the client pc to the side of the server pc in the configuration.

**Comment by axelson** [comment]
Maybe the examples in https://github.com/input-leap/input-leap/discussions/1776 work? I haven't tested them but that previously worked on synergy (and maybe barrier)

**Comment by barrelltitor** [comment]
They work only for dual screen setups because in barrier/synergy/etc the monitor area is just one huge box. With 3+ monitors it won't work unless they're in a box position with an even number of screens i.e

```
+----++----++----+
| A1 || A2 |
+----++----++----+
+----++----++----+
| A3  || A4 |
+----++----++----+
+----++----++----+
| B1 |  | B2 |
+----++----++----+
```
However, this is currently impossible:
```
+----++----++----+
| A1 || A2 |
+----++----++----+
+----++----++----+
| A3  || B |
+----++----++----+
```

You can only go:

A2 Right -> B
A1 Left -> B 
I assume you can also go A1 and A2 up but I haven't tested

You cannot go down, and you cannot go 
A3 Right -> B 
A2 Down -> B
